recipes/libassert/all/conanfile.py

```diff
@@ -105,14 +126,21 @@ def package_info(self):
 
         self.cpp_info.set_property("cmake_file_name", "assert")
         self.cpp_info.set_property("cmake_target_name", "assert::assert")
+        
+        # For some reason after conan installs the include is #include <assert/assert/assert.hpp> which is not what
+        # should happen
+        if Version(self.version) >= Version("1.2.1"):
+            self.cpp_info.includedirs = [os.path.join("include", "assert")]
```

Comment:
Maybe better we append this folder unconditionally and we will not break consumers of the previous versions of the package. Note that this also could be managed patching the original CMakeLists.txt
```suggestion
        self.cpp_info.includedirs.append(os.path.join("include", "assert"))
```

Reply:
Thanks! I had been worried about breaking anything but this seems like a good way to go.